app/controllers/admin_controller.rb

```diff
@@ -39,17 +39,25 @@ def export_str(shf_apps)
       out_str << t("shf_applications.state.#{m_app.state}")
       out_str << ','
 
+      # state date
+      out_str << (m_app.updated_at.strftime('%F'))
+      out_str << ','
+
       # add the business categories, all surrounded by double-quotes
       out_str << '"' + m_app.business_categories.map(&:name).join(', ') + '"'
       out_str << ','
 
       # a company name may have commas, so surround with quotes so spreadsheets recognize it as one string and not multiple comma-separated value
-      out_str << (m_app.companies.empty? ?  '' : "\"#{m_app.companies.last.name}\"")
+      out_str << (m_app.company_id.nil? ?  '' : "\"#{Company.find_by_id(m_app.company_id).name}\"")
```

Comment:
This is wrong.  The PR merged yesterday changed the association between an app and a company(s) from `belongs_to` to `has_and_belongs_to_many`. 

An app represents a member's association with the SHF org, not a specific company.  As part of this, a member can (or will be, when we complete the larger refactoring) be associated with multiple companies via this association: `has_many :companies, through: :shf_applications`.

So, this change should be reverted back to the original statement.

Later, when we've enabled the user<>companies association (1 to N), we'll need to change this logic to also loop through _all_ companies.  But that is not needed now.

Reply:
the original statement does very much not work. 
companies = nil.

Reply:
So, now maybe companies will not be nil, when I have done the db_recreate.

Reply:
Buut.... 
Production wont do db_recreate =/
And my code worked before doing a db_recreate. (where yours did not)